documentation/staging/content/faq/resource-settings.md

```diff
@@ -23,13 +23,63 @@ description: "Tune container memory and CPU usage by configuring Kubernetes reso
+Requests and limits should be configured based on the expected traffic during peak usage.
+For example: 
+
+- Tune CPU and memory high enough
+  to handle expected peak workloads for applications
+  that require large amounts of in-memory processing
+  or very CPU intensive calculations.
+- Tune memory high enough so that WebLogic JMS messaging
+  applications that generate large backlogs of unprocessed
+  persistent or non-persistent messages
+  can expect JMS to efficiently cache the backlogs in memory.
+- CPU requirements are sometimes significantly higher
+  when a WebLogic Server is starting. This means that a low CPU
+  allocation that might be suitable for
+  light runtime workloads risks causing unacceptably slow startup times.
+
+Requirements vary considerably between use cases.
+You may need to experiment and make adjustments
+based on monitoring resource usage in your environment.
+
+The operator creates a container in its own Pod for each domain's
+WebLogic Server instances and for the short lived introspector job that
+is automatically launched before WebLogic Server pods are launched.
+You can tune container memory and CPU usage
+by configuring Kubernetes resource requests and limits,
+and you can tune a WebLogic JVM heap usage
+using the `USER_MEM_ARGS` environment variable in your Domain YAML file.
+The introspector job pod uses the same CPU and memory settings as the
+domain's administration WebLogic Server pod.
+A resource request sets the minimum amount of a resource that a container requires.
+A resource limit is the maximum amount of a resource a container is given
+and prevents a container from using more than its share of a resource.
+Additionally, resource requests and limits determine a Pod's quality of service.
 
 This FAQ discusses tuning these parameters so WebLogic Server instances run efficiently.
 
-#### Setting resource requests and limits in a Domain
-
-You can set Kubernetes memory and CPU requests and limits in a [Domain YAML file]({{< relref "/userguide/managing-domains/domain-resource" >}}) using its `spec.serverPod.resources` stanza, and you can override the setting for individual WebLogic Server instances or clusters using the `serverPod.resources` element in `spec.adminServer`, `spec.clusters`, or `spec.managedServers`. For example:
+#### Setting resource requests and limits in a Domain resource
+
+You can set Kubernetes memory and CPU requests and limits in a
+[Domain YAML file]({{< relref "/userguide/managing-domains/domain-resource" >}})
+using its `spec.serverPod.resources` stanza,
+and you can override the setting for individual WebLogic Server instances or clusters using the
+`serverPod.resources` element in `spec.adminServer`, `spec.clusters`, or `spec.managedServers`.
+Note that the introspector job pod uses the same settings
+as the WebLogic administration Server pod.
+
+Values set in the ".serverPod" stanzas for a more specific type of pod override
```

Comment:
".serverPod" -> `.serverPod` (not sure why you chose to use double-quotes instead of back ticks (code font), but if your intent was to indicate all stanzas matching the pattern `.serverPod`, the back ticks will suffice (or you can use an asterisk to be more explicit, `*.serverPod`)
specific type of pod override -> specific type of pod, (comma) override